mne_bids/write.py

```diff
@@ -933,6 +1000,38 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         bids_raw_folder = bids_fname.split('.')[0]
         bids_fname = op.join(bids_raw_folder, bids_fname)
 
+    # Anonymize
+    if anonymize:
+        if 'daysback' not in anonymize:
+            raise ValueError('`daysback` argument required to anonymize.')
+        daysback = anonymize['daysback']
+        if daysback < 0:
+            raise ValueError('`daysback` must be a positive number')
+        # this is for ctf and set which have meas_date = None in testing data
+        if raw.info['meas_date'] is None:
+            raw.info['meas_date'] = (np.int32(0), np.int32(0))
```

Comment:
you shouldn't introduce a date if there was no date

Reply:
Until Luke pushes his fix, this causes an error with None meas_dates https://github.com/mne-tools/mne-python/pull/7016

Reply:
Then we have to wait for that PR to be merged

Reply:
Might be a while haha we'll see

Reply:
Ok I changed it so that in the tests the set and ctf files are set to meas_date = (0, 0), that way I don't think it's strictly necessary to wait for mne-tools/mne-python#7016 but we can still wait if you want